modules/components/src/DataTable/TableToolbar/helpers.js

```diff
@@ -0,0 +1,67 @@
+import download from '../../utils/download';
+
+export const saveTSV = async ({ url, files = [], fileName, options = {} }) =>
+  download({
+    url,
+    method: 'POST',
+    ...options,
+    params: {
+      fileName,
+      files: files.map(({ allColumns, columns, exColumns = null, ...file }, i) => ({
+        ...file,
+        columns: exColumns // if the component gave you custom columns to show
+          ? Object.values(
+              exColumns.length > 0 // if they ask for any specific columns
+                ? Object.keys(allColumns).filter((fieldName) => exColumns.includes(fieldName)) // use them
+                : allColumns, // else, they're asking for all the columns
+            )
```

Comment:
For future reference, this is where the default columns are given, in case we decide that option doesn't work.

Line 16, particularly, could either have `columns`, to provide only the 'active' columns.